sleap/gui/color.py

```diff
@@ -257,6 +265,29 @@ def get_item_color(
 
             return (128, 128, 128)
 
+        if parent_frame_idx is None and parent_instance:
+            parent_frame = parent_instance.frame
+            if parent_frame:
+                parent_frame_idx = parent_frame.frame_idx
+
+        if parent_session is None and self.labels and parent_frame:
+            parent_session = self.labels.get_session(video=parent_frame.video)
+
+        if (
+            self.distinctly_color == "instance_groups"
+            and parent_session
+            and parent_frame_idx
+        ):
+            frame_group = parent_session.frame_groups.get(parent_frame_idx, None)
+            if frame_group is not None:
+                instance_group = frame_group.get_instance_group(
+                    instance=parent_instance
+                )
+
+            if instance_group is not None:
```

Comment:
We also need to add an initialization for `instance_group` in the case that `frame_group is None`:
```suggestion
            instance_group = None
            frame_group = parent_session.frame_groups.get(parent_frame_idx, None)
            if frame_group is not None:
                instance_group = frame_group.get_instance_group(
                    instance=parent_instance
                )

            if instance_group is not None:
```